l1-contracts/foundry.toml

```diff
@@ -12,4 +12,7 @@ remappings = [
     "forge-std/=../lib/forge-std/src/",
     "murky/=../lib/murky/src/",
     "foundry-test/=test/foundry/",
+    "@openzeppelin/contracts-v4/=../lib/openzeppelin-contracts-v4/contracts/",
+    "@openzeppelin/contracts-upgradeable-v4/=../lib/openzeppelin-contracts-upgradeable-v4/contracts/",
```

Comment:
It's a mistake that we have remapped to ../lib. Just because we have symlinks it's better to just use `lib`

Reply:
but there doesnt seem to be a sym link in `l1-contracts` (maybe removed by mistake)

Reply:
https://github.com/matter-labs/era-contracts/pull/741